examples/tv-app/tv-common/clusters/media-playback/MediaPlaybackManager.cpp

```diff
@@ -298,18 +302,25 @@ bool MediaPlaybackManager::HandleActivateAudioTrack(const chip::CharSpan & track
 bool MediaPlaybackManager::HandleActivateTextTrack(const chip::CharSpan & trackId)
 {
     bool foundMatch = false;
+    std::string idString(trackId.data(), trackId.size());
+    ChipLogError(Controller, "HandleActivateTextTrack - Looking for %s", idString.c_str());
     for (auto const & availableTextTrack : mAvailableTextTracks)
     {
-        if (strcmp(availableTextTrack.id.data(), trackId.data()) == 0)
+        std::string nextidString(availableTextTrack.id.data(), availableTextTrack.id.size());
+        ChipLogError(Controller, "HandleActivateTextTrack - next track %s", nextidString.c_str());
+        if (strcmp(nextidString.c_str(), idString.c_str()) == 0)
         {
-            mActiveAudioTrack = availableTextTrack;
-            foundMatch        = true;
+            ChipLogError(Controller, "HandleActivateTextTrack - match");
+            mActiveTextTrack = availableTextTrack;
+            foundMatch       = true;
```

Comment:
same here: foundMatch does not seem needeed as a variable (and if using break the last like should be `return foundMatch` without if/else... but dropping foundMatch completely looks more correct.

Reply:
done